examples/react/src/App.tsx

```diff
@@ -1,49 +1,66 @@
-import {Alice, Bob, EnactedPolicy, getPorterUri, SecretKey} from '@nucypher/shared';
+import {
+  Alice,
+  Bob,
+  EnactedPolicy,
+  getPorterUri,
+  initialize,
+  SecretKey,
+  toHexString,
+} from '@nucypher/shared';
 import { ethers } from 'ethers';
-import React from 'react';
 import { useEffect, useState } from 'react';
 
-declare let window: any;
-
-function toHexString(byteArray: Uint8Array) {
-  return Array.from(byteArray, function (byte) {
-    return ('0' + (byte & 0xff).toString(16)).slice(-2);
-  }).join('');
-}
-
-export function App() {
-  const [provider, setProvider] = useState(undefined as ethers.providers.Web3Provider | undefined);
-  const [alice, setAlice] = useState(undefined as Alice | undefined);
-  const [bob, setBob] = useState(undefined as Bob | undefined);
-  const [policy, setPolicy] = useState(undefined as EnactedPolicy | undefined);
+function App() {
+  const [isInit, setIsInit] = useState<boolean>(false);
+  const [provider, setProvider] = useState<
+    ethers.providers.Web3Provider | undefined
+  >();
+  const [alice, setAlice] = useState<Alice | undefined>();
+  const [bob, setBob] = useState<Bob | undefined>();
+  const [policy, setPolicy] = useState<EnactedPolicy>();
+
+  const initNucypher = async () => {
```

Comment:
This is an internal function, right? Name is not very descriptive, but if it's only a local thing, it's fine.

Reply:
Yes, it's only used in this example as a helper method